### ISSUE TITLE
btrfs-progs v5.7: fsck-tests/037-freespacetree-repair, convert-tests/017-fs-near-full, cli-tests/014-multiple-profiles-warning tests fail (ppc)

### ISSUE THREAD

**Issue by ernsteiswuerfel** [kdave/btrfs-progs#262]
Just did a testrun with btrfs-progs from git-devel (**11a360b**) on my G4, kernel 5.8-rc2.

These two tests fail:
```
fsck-tests/037-freespacetree-repair
convert-tests/017-fs-near-full
```
[fsck-tests-results.txt](https://github.com/kdave/btrfs-progs/files/4838176/fsck-tests-results.txt)
[convert-tests-results.txt](https://github.com/kdave/btrfs-progs/files/4838178/convert-tests-results.txt)

**Comment by marcosps** [comment]
@kdave I can reproduce the same fsck-test problem here, and indeed, there are no FREE_SPACE_EXTENT items in my fs. But this patch[1] from @asj  says that sometimes we don't have a FREE_SPACE_BITMAP. Maybe it's the same case here? The documentation about FREE_SPACE_EXTENT is pretty vague in btrfs-dev-docs[2].

[1]: https://lore.kernel.org/linux-btrfs/1585879843-17731-2-git-send-email-anand.jain@oracle.com/#t
[2]: https://github.com/btrfs/btrfs-dev-docs/blob/master/tree-items.txt#L796

**Comment by ernsteiswuerfel** [comment]
Re-test with btrfs-progs v5.7 and kernel 5.8-rc5. Forgot to mention that `cli-tests/014-multiple-profiles-warning` fails too just like on x86 (issue #251 )
[cli-tests-results.txt](https://github.com/kdave/btrfs-progs/files/4928345/cli-tests-results.txt)

**Comment by ernsteiswuerfel** [comment]
Closed in favour of the more generic issue #251 which was reported earlier.
